server/controllers/questions.js

```diff
@@ -29,6 +30,46 @@ exports.getAnswers = (req, res) =>{
   });
 };
 
+// exports.getQuestions = (req, res) =>{
+//   let apiGetQ = 'https://app-hrsei-api.herokuapp.com/api/fec2/hr-rpp/qa/questions?product_id=';
+//   axios.get(apiGetQ+'71698', {
+//   headers: {
+//     'Authorization': process.env.API_KEY
+//   }
+//   })
+//   .then((result)=>{
+//     // console.log(result.data);
+//     res.send(result.data);
+//   });
+// };
+
+exports.addQuestion = (req, res) =>{
+  let apiGetQ = 'https://app-hrsei-api.herokuapp.com/api/fec2/hr-rpp/qa/questions/';
```

Comment:
Another advantage to getting the URL from an .env file is not having it hard-coded in multiple places